Quieter message to failed Lua engine

### Description

Instead of logging the full stack trace:

```
2026-03-04 18:00:25,944 WARN  [o.i.s.r.RespServer] Failed to load script engine. Script features disabled for RESPjava.lang.LinkageError: Unable to find natives or init
    at party.iroiro.luajava.lua51.Lua51.getNatives(Lua51.java:59)
    at party.iroiro.luajava.lua51.Lua51.<init>(Lua51.java:46)
    at org.infinispan.server.resp.scripting.LuaContext.<init>(LuaContext.java:84)
    at org.infinispan.server.resp.scripting.LuaContextPool.<init>(LuaContextPool.java:19)
    at org.infinispan.server.resp.scripting.LuaTaskEngine.<init>(LuaTaskEngine.java:41)
    at org.infinispan.server.resp.RespServer.initializeLuaTaskEngine(RespServer.java:162)
    at org.infinispan.server.resp.RespServer.internalPostStart(RespServer.java:79)
    at org.infinispan.server.core.AbstractProtocolServer.postStart(AbstractProtocolServer.java:149)
    at java.base/java.util.concurrent.ConcurrentHashMap$ValuesView.forEach(ConcurrentHashMap.java:4800)
    at org.infinispan.server.SecurityActions.lambda$postStartProtocolServer$0(SecurityActions.java:68)
    at org.infinispan.security.Security.doPrivileged(Security.java:62)
    at org.infinispan.server.SecurityActions.postStartProtocolServer(SecurityActions.java:67)
    at org.infinispan.server.Server.lambda$run$2(Server.java:521)
    at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:884)
    at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:862)
    at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:531)
    at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2206)
    at org.infinispan.server.Server.lambda$startCacheManager$0(Server.java:553)
    at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1825)
    at java.base/java.util.concurrent.ThreadPerTaskExecutor$TaskRunner.run(ThreadPerTaskExecutor.java:291)
    at java.base/java.lang.VirtualThread.run(VirtualThread.java:456)
Caused by: java.lang.IllegalStateException: com.badlogic.gdx.utils.SharedLibraryLoadRuntimeException: Couldn't load shared library 'liblua5164.so' for target: Linux, x86, 64-bit
    at party.iroiro.luajava.lua51.Lua51Natives.<init>(Lua51Natives.java:129)
    at party.iroiro.luajava.lua51.Lua51.getNatives(Lua51.java:57)
    ... 20 more
Caused by: com.badlogic.gdx.utils.SharedLibraryLoadRuntimeException: Couldn't load shared library 'liblua5164.so' for target: Linux, x86, 64-bit
    at com.badlogic.gdx.utils.SharedLibraryLoader.load(SharedLibraryLoader.java:174)
    at party.iroiro.luajava.util.GlobalLibraryLoader.load(GlobalLibraryLoader.java:63)
    at party.iroiro.luajava.lua51.Lua51Natives.<init>(Lua51Natives.java:123)
    ... 21 more
Caused by: com.badlogic.gdx.utils.SharedLibraryLoadRuntimeException: Unable to read file for extraction: liblua5164.so
    at com.badlogic.gdx.utils.SharedLibraryLoader.readFile(SharedLibraryLoader.java:183)
    at com.badlogic.gdx.utils.SharedLibraryLoader.loadFile(SharedLibraryLoader.java:339)
    at com.badlogic.gdx.utils.SharedLibraryLoader.load(SharedLibraryLoader.java:170)
    ... 23 more
```

Log the message warning and the stack trace in debug.

### Implementation ideas

_No response_

--- a/server/resp/src/main/java/org/infinispan/server/resp/RespServer.java
+++ b/server/resp/src/main/java/org/infinispan/server/resp/RespServer.java
@@ -166,7 +166,8 @@ public class RespServer extends AbstractProtocolServer<RespServerConfiguration> 
          TaskManager taskManager = gcr.getComponent(TaskManager.class);
          taskManager.registerTaskEngine(luaTaskEngine);
       } catch (Exception | LinkageError e) {
-         log.failedToLoadScriptEngine(e);
+         log.failedToLoadScriptEngine();
+         log.debugf(e, "Failed to register script engine for RESP cache");
       }
    }
 

--- a/server/resp/src/main/java/org/infinispan/server/resp/logging/Log.java
+++ b/server/resp/src/main/java/org/infinispan/server/resp/logging/Log.java
@@ -79,6 +79,6 @@ public interface Log extends BasicLogger {
    void enableTransactionForMultiExec();
 
    @LogMessage(level = WARN)
-   @Message(value = "Failed to load script engine. Script features disabled for RESP")
-   void failedToLoadScriptEngine(@Cause Throwable cause);
+   @Message(value = "Failed to load script engine. Script features disabled for RESP", id = 13012)
+   void failedToLoadScriptEngine();
 }

